ca/ca.go

```diff
@@ -86,6 +85,10 @@ type certificateAuthorityImpl struct {
 	orphanCount        *prometheus.CounterVec
 	adoptedOrphanCount *prometheus.CounterVec
 	signErrorCount     *prometheus.CounterVec
+
+	// This is temporary, and will be used for testing and slow roll-out
+	// of ECDSA issuance, but will then be removed.
+	ecdsaAllowList *ECDSAAllowList
```

Comment:
Despite being temporary, I'd put this higher, like up where the old `ecdsaAllowedRegIDs` entry was. In general, large collections of items like this should be ordered in a semantically meaningful way, e.g. by relevance or importance, rather than just by order of introduction. Here, the log and various metrics are non-functional (useful, but don't affect the behavior of the program), and the clk is test-only, so those should always come last.